lib/vanagon/driver.rb

```diff
@@ -76,14 +76,13 @@ def run
       @engine.startup(@workdir)
 
       puts "Target is #{@engine.target}"
-
-      install_build_dependencies
+      retry_task { install_build_dependencies }
       @project.fetch_sources(@workdir)
       @project.make_makefile(@workdir)
       @project.make_bill_of_materials(@workdir)
       @project.generate_packaging_artifacts(@workdir)
       @engine.ship_workdir(@workdir)
-      @engine.dispatch("(cd #{@engine.remote_workdir}; #{@platform.make})")
+      retry_task { @engine.dispatch("(cd #{@engine.remote_workdir}; #{@platform.make})") }
```

Comment:
For right now, I think it's safe to say that we want this everywhere, at least the two tasks that are currently targeted. In the future, I think adding by-engine and/or by-project configuration knobs makes sense though.

As it stands, I don't think it hurts to have (if you don't want retries set them to '1' in the project) and it's a good place to start experimenting and see where to go next.